packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -0,0 +1,1200 @@
+import React from 'react';
```

Comment:
This file is too big.
![](https://media.giphy.com/media/kZDGNBLCIOTmw/giphy.gif)

Reply:
I moved the statics out into their own utility file, which cut out a couple hundred lines. I'll look at some of the other functions to see what could get extracted easily.